master/static/srv/command-entrypoint.sh

```diff
@@ -10,6 +10,10 @@ set -e
 # to register the proxy with the Determined master.
 "$DET_PYTHON_EXECUTABLE" -m determined.exec.prep_container --proxy --download_context_directory
 
+set -x
```

Comment:
do commands not support start up hook?

maybe just make a ticket calling this out? Not sure if that is intentional

Reply:
https://hpe-aiatscale.atlassian.net/browse/RM-159